src/Pages/PostWork/index.js

```diff
@@ -1,50 +1,276 @@
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import moment from "moment";
+
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
 
 import Form from "react-bootstrap/Form";
-import Container from "react-bootstrap/Container";
+
 import Button from "react-bootstrap/Button";
 import Col from "react-bootstrap/Col";
+import Image from "react-bootstrap/Image";
+
+import { submitPost } from "../../store/post/actions";
+
+import { Formik } from "formik";
+import * as Yup from "yup";
+import "./PostWork.css";
+
+// Schema for yup
+const validationSchema = Yup.object().shape({
+  title: Yup.string()
+    .min(4, "*Post title must have at least 4 characters")
+    .max(25, "*Post title can't be longer than 25 characters")
+    .required("*Title is required"),
+  description: Yup.string()
+    .min(10, "*Description must have at least 10 characters")
+    .required("*Description is required"),
+  imageUrl: Yup.string()
+    .url("*Must enter URL in http://www.example.com format")
+    .required("*Image URL required"),
+  myDate: Yup.date().required("*Date is required").min(new Date()),
+});
 
 export default function PostWork() {
-  const [title, setTitle] = useState("");
-  const [description, setDescription] = useState("");
+  const [fromMin, setFromMin] = useState();
+  const [toMin, setToMin] = useState();
+
+  const today = new Date();
+
+  const dispatch = useDispatch();
+
+  const [myDate, setDate] = useState(today);
+  const [fromTime, setFromTime] = useState(today);
+  const [toTime, setToTime] = useState();
+
+  useEffect(() => {
+    const calculateFromTime = () => {
+      let minFromTime;
+      if (myDate) {
+        if (
+          moment(myDate).format("YYYY-MM-DD") ===
+          moment(new Date()).format("YYYY-MM-DD")
```

Comment:
`moment(myDate).isSame(new Date())` or `moment(myDate).isSame(moment())` take advantage of all of moment's cool methods 😄